### DISCUSSION TITLE
LOLIN D32 Pro with LOLIN TFT-2.4 and TFT cable

### DISCUSSION THREAD

**Discussion by WhyDoYouMakeUsDoThis** [HASwitchPlate/openHASP#254] [Hardware]
Might be answered elsewhere... Sorry I can't find it.
I have the D32 Pro and LOLIN 2.4 display connected by the 10 pin TFT cable.
I've loaded the d1-mini-esp32_ili9341_ota_v0.6.2.bin on the board.
OpenHASP works fine, I have it connected to the WiFi.. But the display is white with nothing else.
I'm wondering if the default pins for the display are not correct for the TFT cable.
Is there a way of correcting it without recompling or what values do I need to change to rebuild a new image?

Thanks

**Comment by fvanroie** [comment]
You need the firmware [`lolin-d32-pro_ili9341_full_16MB_v0.6.2.bin`](https://github.com/HASwitchPlate/openHASP/releases/tag/v0.6.2) for the correct pin-out.
The pre-compiled binary was not yet included in the v0.6.2 release until now. I have just added that configuration to the v0.6.2 release.
If you upload this firmware to your device the screen should work out of the box.

**Reply by WhyDoYouMakeUsDoThis** [reply]
Thanks!  That firmware works right away!
